pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -490,7 +490,7 @@ func (tc *testCase) runTest(t *testing.T) {
 	)
 
 	eventClient := &clientfake.Clientset{}
-	eventClient.AddReactor("*", "events", func(action core.Action) (handled bool, ret runtime.Object, err error) {
+	eventClient.AddReactor("create", "events", func(action core.Action) (handled bool, ret runtime.Object, err error) {
 		tc.Lock()
```

Comment:
I'm wondering how complicated it will be to add an extensive unit test that checks if the newly added rate limiting works correctly.